src/main/java/org/opentripplanner/model/impl/OtpTransitServiceBuilder.java

```diff
@@ -64,17 +69,15 @@
 
     private final EntityById<FeedScopedId, Trip> tripsById = new EntityById<>();
 
-    private final Multimap<StopPattern, TripPattern> tripPatterns = HashMultimap.create();
+    private final Multimap<StopPattern, TripPattern> tripPatterns = ArrayListMultimap.create();
```

Comment:
Storing TripPattern in a Set will fail during GTFS import because the ids (code) is calculated in the PatternHopFactory - after TripPatterns are added here. Hence; the TripPattern#hashCode() does not work properly. This worked before because TripPattern did not implement the hashCode() function, instead used the inherited one from Object.